coc/abc.py

```diff
@@ -50,48 +53,52 @@ class BaseClan:
     level: :class:`int`
         The clan's level.
     """
+    __slots__ = ("tag", "name", "_client", "badge", "level", "_response_retry", "_raw_data")
 
-    __slots__ = ("tag", "name", "_client", "badge", "level", "_response_retry")
+    def __init__(self, *, data, client, **_):
```

Comment:
I understand the need to use `**_` since the kwargs are not used. But we should stick to python norms and use `**kwargs` in the signature and leave the anonymous symbol for loops like `for _ in container:`